_msprimemodule.c

```diff
@@ -5194,6 +5224,23 @@ SparseTree_get_population(SparseTree *self, PyObject *args)
     return ret;
 }
 
+static PyObject *
```

Comment:
We don't need this I think. The ``population`` getter is left over from the coalescence-record days. We can get the individual for a given node ID from the tree sequence, so we don't need this method.

Reply:
Sounds good. It's a bit hard for me to see how the python-C glue is structured, so it's good you're having a pass.